config/samples/workloads/deployment_definition.yaml

```diff
@@ -7,29 +7,31 @@ spec:
     name: deployments.apps
   extension:
     alias: deployment
-    object:
-      apiVersion: core.oam.dev/v1alpha2
-      kind: deployment
-      metadata:
-        name: tbd
-      spec:
-        containers:
-          - image: myrepo/myapp:v1
-            name: master
-            ports:
-              - containerPort: 6379
-                protocol: TCP
-                name: tbd # TODO(zzxwill) A temporary workaround for ContainerizedWorkload
-    parameters:
-      - name: image
-        short: i
-        required: true
-        type: string
-        fieldPaths:
-          - "spec.containers[0].image"
-      - name: port
-        short: p
-        required: false
-        type: int
-        fieldPaths:
-          - "spec.containers[0].ports[0].containerPort"
+    template: |
```

Comment:
It is better to put cue template into a separate file.
We can use cli to specify the .cue file for a definition.

Reply:
next PR